demos/widgets/lv_demo_widgets.c

```diff
@@ -1504,14 +1504,8 @@ static void chart_event_cb(lv_event_t * e)
             if(lv_chart_get_type(obj) == LV_CHART_TYPE_BAR) {
```

Comment:
After the change, it seems that there is no need to determine whether bar or line, if and else content is exactly the same